### PR TITLE
Cluster group volumes backup

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
This PR documents the new ability users have to backup the disks of a virtual cluster.
💻 [Preview](https://deploy-preview-1140--docs-spectrocloud.netlify.app/clusters/cluster-groups/cluster-group-backups)
🎫 [PPD-1116](https://spectrocloud.atlassian.net/browse/PPD-1116)

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1140#" title="Author self-approved">karl-cardenas-coding</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1140#" title="Author self-approved">karl-cardenas-coding</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-2/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/04-clusters/06-cluster-management/05-backup-restore.md
Comment (ritawatson) [inline content/docs/04-clusters/06-cluster-management/05-backup-restore.md:263]:
```suggestion
Backups can be scheduled or initiated on demand during cluster creation. Backups can also be scheduled for a running cluster. The following information is required to configure a cluster backup:
```

File: content/docs/04-clusters/20-cluster-groups/25-cluster-group-backups.md
Comment (ritawatson) [inline content/docs/04-clusters/20-cluster-groups/25-cluster-group-backups.md:4]:
```suggestion
metaDescription: "Learn how to configure disk and volume backup for virtual clusters in a cluster group."
```

File: content/docs/04-clusters/20-cluster-groups/25-cluster-group-backups.md
Comment (ritawatson) [inline content/docs/04-clusters/20-cluster-groups/25-cluster-group-backups.md:17]:
```suggestion
Palette [Virtual Clusters](/clusters/palette-virtual-clusters) are a capability that cluster groups support and that you can enable when creating a cluster group. By default, the virtual cluster settings in a cluster group disable disk backups. You can back up all the volumes within a virtual cluster using the following steps. 
```

File: content/docs/04-clusters/20-cluster-groups/25-cluster-group-backups.md
Comment (ritawatson) [inline content/docs/04-clusters/20-cluster-groups/25-cluster-group-backups.md:43]:
```suggestion
3. Select a cluster group to enable virtual cluster disk backup.
```

File: content/docs/04-clusters/20-cluster-groups/25-cluster-group-backups.md
Comment (ritawatson) [inline content/docs/04-clusters/20-cluster-groups/25-cluster-group-backups.md:46]:
```suggestion
4. Click **Settings** and expand the **Settings** Menu.
```

Comment (ritawatson) [reply content/docs/04-clusters/20-cluster-groups/25-cluster-group-backups.md:46]: Let's combine steps 4 and 5.

File: content/docs/04-clusters/20-cluster-groups/25-cluster-group-backups.md
Comment (ritawatson) [inline content/docs/04-clusters/20-cluster-groups/25-cluster-group-backups.md:87]:
```suggestion
1. Log in to [Palette](https://console.spectrocloud.com).
```

File: content/docs/04-clusters/20-cluster-groups/25-cluster-group-backups.md
Comment (ritawatson) [inline content/docs/04-clusters/20-cluster-groups/25-cluster-group-backups.md:93]:
```suggestion
3. Create a backup of your virtual cluster and include all disks. Use the [Create a Cluster Backup](/clusters/cluster-management/backup-restore#createaclusterbackup) guide for additional guidance.
```

File: content/docs/04.5-devx/05-app-profile/08-services/15-db-services.md
Comment (ritawatson) [inline content/docs/04.5-devx/05-app-profile/08-services/15-db-services.md:46]:
```suggestion
By default, cluster groups are configured not to back up the disk storage. This default behavior affects database services because a backup would not include the storage disk. To learn more, refer to [Set Up Disk Backup for Virtual Clusters](/clusters/cluster-groups/cluster-group-backups).
```

File: content/docs/04-clusters/04-palette-virtual-clusters/02-deploy-virtual-cluster.md
Comment (ritawatson) [inline content/docs/04-clusters/04-palette-virtual-clusters/02-deploy-virtual-cluster.md:65]:
```suggestion
Follow these steps to deploy a virtual cluster to a cluster group:
```

Comment (ritawatson) [reply content/docs/04-clusters/04-palette-virtual-clusters/02-deploy-virtual-cluster.md:65]: Or, to use the plural: Follow these steps to deploy virtual clusters to a cluster group:

File: content/docs/04-clusters/04-palette-virtual-clusters/02-deploy-virtual-cluster.md
Comment (ritawatson) [inline content/docs/04-clusters/04-palette-virtual-clusters/02-deploy-virtual-cluster.md:76]:
```suggestion
4. Select your cluster group from the **Select cluster group drop-down Menu**, and type a name for the virtual cluster.
```

File: content/docs/04-clusters/04-palette-virtual-clusters/02-deploy-virtual-cluster.md
Comment (ritawatson) [inline content/docs/04-clusters/04-palette-virtual-clusters/02-deploy-virtual-cluster.md:73]:
```suggestion
3. Click the **+New Virtual Cluster**.
```

Comment (ritawatson) [reply content/docs/04-clusters/04-palette-virtual-clusters/02-deploy-virtual-cluster.md:73]: Are we including the + in the button name? I've been using only the text label. As a user, I see the + as a visual indicator that I can add something, but to call it by name I'd say **New Virtual Cluster** button or click **New Virtual Cluster.** Maybe we need to discuss and add to the internal style guidelines.

**Review by ritawatson — APPROVED** [review]
LGTM with some comments.

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1140#" title="Author self-approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1140#pullrequestreview-1304077961" title="Approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-2/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/04-clusters/04-palette-virtual-clusters/02-deploy-virtual-cluster.md
Comment (karl-cardenas-coding) [reply content/docs/04-clusters/04-palette-virtual-clusters/02-deploy-virtual-cluster.md:65]: No, you're right. It should be singular, nice catch 😄

File: content/docs/04-clusters/04-palette-virtual-clusters/02-deploy-virtual-cluster.md
Comment (karl-cardenas-coding) [reply content/docs/04-clusters/04-palette-virtual-clusters/02-deploy-virtual-cluster.md:73]: Yeah, let's chat. I'll add it to the team agenda

**Pull request merged**
